### PR TITLE
Skip compiler activation (on Linux) for 4.0.x

### PR DESCRIPTION
**Pull request by jakirkham**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Fixes https://github.com/conda-forge/ctng-compiler-activation-feedstock/issues/74

Backports PR ( https://github.com/conda-forge/r-base-feedstock/pull/213 ) to `4.0.x`.

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,7 +28,8 @@ source:
     - 0018-Use-LAPACK_LDFLAGS-in-Rlapack_la_LIBADD.patch
 
 build:
-  number: 6
+  skip: true  # [win]
```
Comment (isuruf) [inline recipe/meta.yaml:31]: Why? These build fine on windows.

Comment (jakirkham) [inline recipe/meta.yaml:31]: Note [Windows is already disabled in `main`]( https://github.com/conda-forge/r-base-feedstock/blob/6a79d65c6813758360bd884020ac1106471cf470/recipe/meta.yaml#L31 ) due to issue ( https://github.com/conda-forge/r-base-feedstock/issues/214 ). Given this issue would be seen with any Windows build of `r-base`, went ahead and just disabled here until that issue can be resolved.

**Comment by jakirkham** [comment]
Thanks Devon! 🙏

**Pull request merged**